_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -105,20 +114,22 @@ To set up an alert for a detector, continue with the following steps:
 
     <img src="{{site.url}}{{site.baseurl}}/images/Security/rule_name_delete.png" alt="Deletes all selected rules" width="45%">
 
-    * Select one or more rule severities as conditions for the alert.
+    * Select one or more rule severity levels as conditions for the alert.
     * Select from a list of tags to include as conditions for the alert.
 
 1. To define a notification for the alert, assign an alert severity, select a channel for the notification, and customize a message generated for the alert.
```

Comment:
The image should show examples, then you could say, "The following image shows a trigger named X that will run when the severity is X and has X tag applied."